vllm/model_executor/models/phimoe.py

```diff
@@ -429,6 +430,7 @@ def forward(
         return hidden_states, residual
 
 
+@support_torch_compile
 class PhiMoEModel(nn.Module):
```

Comment:
for this model, it seems directly running it with `-tp=2` will fail. the error is:

`Failed: Cuda error /workspace/csrc/custom_all_reduce.cuh:336 'invalid argument'`

need to investigate it later.

Reply:
note: this is unrelated to `torch.compile`